src/mainwindow.cpp

```diff
@@ -308,12 +308,12 @@ void MainWindow::showConsoleContextMenu(const QPoint &pt)
 
 void MainWindow::closeEvent(QCloseEvent *event)
 {
-    QMessageBox::StandardButton ret = QMessageBox::question(this, "Iaito",
+    QMessageBox::StandardButton ret = QMessageBox::question(this, "iaito",
                  "Do you really want to exit?\nSave your project before closing!",
                  QMessageBox::Ok | QMessageBox::Cancel);
     //qDebug() << ret;
     if (ret == QMessageBox::Ok) {
-        QSettings settings("Radare Corp", "Iaito");
+        QSettings settings("iaito", "iaito");
```

Comment:
Both values should be different, as we are not a company I used "Radare Corp" as a joke :P But good for me using "iaito" in the second value :)